app/models/card.rb

```diff
@@ -10,12 +10,24 @@ class Card < ActiveRecord::Base
   mount_uploader :image, ImageUploader
 
   def set_default_review_date
-    self.review_date = self.review_date + 3.day
+    self.review_date = DateTime.now.to_formatted_s(:iso8601)
   end
 
   def check_translation(input_text)
-    if prepare_text(translated_text) == prepare_text(input_text)
-      self.update_attributes(review_date: (DateTime.current+3.day).to_date)
+    if prepare_text(translated_text) == prepare_text(input_text) && self.accuracy <= -3
+      self.accuracy = 0
+      self.attempt = 1
+      update_attributes(review_date: (self.review_date + calculate_review_date))
+      true
+    elsif prepare_text(translated_text) == prepare_text(input_text) && self.accuracy > -3
+      self.attempt += 1
+      update_attributes(
+          review_date: (self.review_date + calculate_review_date))
```

Comment:
Indent the first parameter one step more than the previous line.<br>Redundant `self` detected.